department-of-veterans-affairs/va.gov-cms
# VACMS-12708: translation access

## Description

Relates to #12708

## QA steps

What needs to be checked to prove this works?  

1. As administrator, visit `/admin/people/roles`.
- [x] A role 'Translation manager' exists.
2. As administrator, visit `/admin/people/permissions`. Test the following permission assignments:

|   |Translation manager|Content admin|
|---|---|---|
|Create translations| ✅  | ✅  |
|Edit translations|✅ |✅ | 
|Translate any entity|✅ |✅ |
|Translate interface text|✅ ||
|Translate configuration|✅ ||

- [x] The permissions defined above are assigned to the roles correctly.
4. Create a new user and give it the 'Translation manager' role.
- [x] This user should have access to `/admin/config/regional/translate`
- [x] This user should have access to `/admin/config/regional/config-translation`
5. Log in as a user with the 'Content admin' role, for example `megan.zehnder@va.gov`.
- [x] This user should not have access to `/admin/config/regional/translate`
- [x] This user should not have access to `/admin/config/regional/config-translation`

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
